applications/sandbox/systems/final_presentation_system.hpp

```diff
@@ -0,0 +1,267 @@
+
+        //bindToEvent<uniform_grid_broad_phase, &final_presentation_system::setUniformGrid>();
+
+        //Add the update function to the update loop.
+        createProcess<&final_presentation_system::update>("Update");
+
+        rendering::model_handle floor;
+        //rendering::model_handle plane;
+        //rendering::model_handle terminal;
+        rendering::model_handle house;
+        rendering::model_handle cube;
+        rendering::model_handle realCube;
+        rendering::material_handle textureH;
+
+        std::vector<gfx::material_handle> materials;
+        gfx::material_handle floor_material;
+        //gfx::material_handle plane_material;
+        //gfx::material_handle terminal_material;
+        gfx::material_handle house_material;
+        gfx::material_handle cube_material;
+
+        //Light stuff
+        ecs::entity_handle directional_light = createEntity();
+        directional_light.add_components<transform>(position(), rotation::lookat(math::vec3(1, 1, 1), math::vec3::zero), scale());
+        directional_light.add_component(gfx::light::directional(math::color(1.f, 1.f, 0.8f), 10.f));
+
+        app::window window = m_ecs->world.get_component_handle<app::window>().read();
+        {
+            app::context_guard guard(window);
+
+            floor = rendering::ModelCache::create_model("floor_final", "assets://models/final_presentation/floor5.glb"_view, materials);
+            floor_material = materials[0];
+            floor_material.set_param("skycolor", math::color(0.1f, 0.3f, 1.f));
+
+            house = rendering::ModelCache::create_model("house_final", "assets://models/final_presentation/2story_1.glb"_view, materials);
+            house_material = materials[1];
+            house_material.set_param("skycolor", math::color(0.1f, 0.3f, 1.f));
+
+            cube = rendering::ModelCache::create_model("cube_final", "assets://models/final_presentation/house_1.glb"_view, materials);
+            cube_material = materials[2];
+            cube_material.set_param("skycolor", math::color(0.1f, 0.3f, 1.f));
+
+            textureH = rendering::MaterialCache::create_material("texturee", "assets://shaders/texture.shs"_view);
+            textureH.set_param("_texture", rendering::TextureCache::create_texture("assets://textures/split-test.png"_view));
+
+            realCube = rendering::ModelCache::create_model("cubee", "assets://models/cube.obj"_view);
+
+            /*plane = rendering::ModelCache::create_model("plane_final", "assets://models/final_presentation/plane.glb"_view, materials);
+            plane_material = materials[1];
+            plane_material.set_param("skycolor", math::color(0.1f,0.3f,1.f));*/
+
+            /*terminal = rendering::ModelCache::create_model("terminal_final", "assets://models/final_presentation/terminal.glb"_view, materials);
+            terminal_material = materials[2];
+            terminal_material.set_param("skycolor", math::color(0.1f, 0.3f, 1.f));*/
```

Comment:
If you do not require these rendering models anymore, perhaps you can remove the code